build.fsx

```diff
@@ -1,11 +1,5 @@
-System.IO.Directory.SetCurrentDirectory __SOURCE_DIRECTORY__
-
-// --------------------------------------------------------------------------------------
-// FAKE build script
-// --------------------------------------------------------------------------------------
-
-#I @"packages/FAKE/tools"
-#r @"packages/FAKE/tools/FakeLib.dll"
+#r "paket: groupref netcorebuild //"
```

Comment:
As I understand, this syntax never got merged?

Reply:
What do you mean?

Reply:
I thought the pull request to add paket references to fsx didn't get merged?

Reply:
Yes this PR is still open. I guess we should do another update of FAKE at this point.